memory/ndctl.py

```diff
@@ -305,6 +306,34 @@ def test_multiple_namespaces_region(self):
             self.create_namespace(region=region, mode='fsdax', size=size / cnt)
             self.log.info("Namespace %s created", nid + 1)
 
+    def test_namespace_reconfigure(self):
+        """
+        Test namespace reconfiguration
+        """
+        self.enable_region()
+        region = self.get_json(short_opt='-R')[0]
+        self.log.info("Using %s for reconfiguring namespace", region)
+        self.disable_namespace()
+        self.destroy_namespace()
+        self.create_namespace(region=region, mode='fsdax', align='64k')
+        old_ns = self.get_json()[0]
+        old_ns_dev = self.get_json_val(old_ns, 'dev')
+        self.log.info("Re-configuring namespace %s", old_ns_dev)
+        self.create_namespace(region=region, mode='fsdax',
+                              name='test_ns', reconfig=old_ns_dev, force=True)
+        new_ns = self.get_json()[0]
+        self.log.info("Checking namespace changes")
+        failed_vals = []
+        for key, val in new_ns.iteritems():
+            if key not in ['uuid', 'dev']:
+                if key in old_ns.keys():
```

Comment:
if key in list(set(old_ns.keys()) - set(['uuid', 'dev'])):

Reply:
:+1: Thanks. Pushed with the change.